Homepage.html

```diff
@@ -79,92 +79,88 @@ <h3>Mentalist Show</h3>
-</div>
-</div>
-<p><a href="https://google.com">Read more reviews</a></p>
+      <span class="yelp-review" data-review-id="Nkl_Usm2UFZEliB-5VfofA" data-hostname="www.yelp.com">Read
+        <a href="https://www.yelp.com/user_details?userid=01P32tHaXUfRWXXeRXvvFA" rel="nofollow noopener">Amanda L.</a>'s
+        <a href="https://www.yelp.com/biz/magician-david-gabbay-los-angeles-2?hrid=Nkl_Usm2UFZEliB-5VfofA" rel="nofollow noopener">review</a> of
+        <a href="https://www.yelp.com/biz/ybr--yehcKCXwM6u6seTqQ" rel="nofollow noopener">Magician David Gabbay</a> on
+        <a href="https://www.yelp.com" rel="nofollow noopener">Yelp</a>
+      </span>
+    </div>
+    <div class="yelp-container">
+      <span class="yelp-review" data-review-id="xDi0zdKcEg5LpSwNrm3uDA" data-hostname="www.yelp.com">Read
+        <a href="https://www.yelp.com/user_details?userid=Bkv9SeDIAJ_X5-NeZCdF3A" rel="nofollow noopener">Alice L.</a>'s
+        <a href="https://www.yelp.com/biz/magician-david-gabbay-los-angeles-2?hrid=xDi0zdKcEg5LpSwNrm3uDA" rel="nofollow noopener">review</a> of
+        <a href="https://www.yelp.com/biz/ybr--yehcKCXwM6u6seTqQ" rel="nofollow noopener">Magician David Gabbay</a> on
+        <a href="https://www.yelp.com" rel="nofollow noopener">Yelp</a>
+      </span>
+    </div>
+  </div>
+  <p>
+    <a href="https://google.com">Read more reviews</a>
+  </p>
 </section>
 
 <section class="footer">
   <div>
-  <p>David Gabbay Magic</p>
+    <p>David Gabbay Magic</p>
   </div>
   <div>
-  <ul>
-    <li><a href="meet-david.html">Meet David</a></li>
-    <li><a href="shows.html">Shows</a></li>
-    <li><a href="Reviews">Reviews</a></li>
-    <li><a href="https://google.com">Contact</a></li>
-  </ul>
-</div>
-</section>
-
-<!--Small viewport footer-->
-<section class="mobile-footer">
-  <div>
-  <p>David Gabbay Magic</p>
+    <ul>
+      <li><a href="meet-david.html">Meet David</a></li>
+      <li><a href="shows.html">Shows</a></li>
+      <li><a href="Reviews">Reviews</a></li>
+      <li><a href="https://google.com">Contact</a></li>
+    </ul>
   </div>
-  <div>
-  <ul>
-    <li><a href="meet-david.html">Meet David</a></li>
-    <li><a href="shows.html">Shows</a><br><br></li>
-    <li><a href="Reviews">Reviews</a></li>
-    <li><a href="https://google.com">Contact</a></li>
-  </ul>
-</div>
 </section>
-<!--Small viewport footer-->
 
+<script async="async" src="https://www.yelp.com/embed/widgets.js"></script>
```

Comment:
In almost all cases we load scripts at the bottom. This is because if there are any errors it would prevent the html or anything below it from loading.